guides/Guide to Dream Maker.md

```diff
@@ -0,0 +1,49 @@
+Guide to the Dream Maker
```

Comment:
This should definitely be on the github wiki

Reply:
https://github.com/PolarisSS13/Polaris/wiki/Intro-to-the-Dream-Maker

Reply:
7o
Thank you! I lack perms to add stuff to Wiki, so it's appreciated <3